README.md

```diff
@@ -45,15 +50,32 @@ Note that:
   a shifting element.
 
 * An element whose start location changes by less than 3 CSS pixels is not a
-  shifting element.
+  shifting element.  This threshold was chosen to avoid penalizing trivial
+  movements.  It also allows some animations (but a broader allowance is
+  described in "Transform Changes" below).
+
+### Transform Changes
+
+Changing an element's [transform](https://developer.mozilla.org/en-US/docs/Web/CSS/transform)
+affects its visual representation.  However, because
+
+* transform changes don't reflow surrounding content,
+* transform changes are a common target of fluid animations, and
+* animated transform changes are easily rendered with hardware-accelerated
```

Comment:
Is this a Chrome-specific quality or do all browsers do this?

Reply:
I think all browsers have accelerated compositing for transform animations